### DISCUSSION TITLE
Use a normal license

### DISCUSSION THREAD

**Discussion by icarusinthewings** [CatalystDevOrg/Catalyst#48] [General]
Please use a normal license, or I will not give permission for my logo to be used with Catalyst.
The MIT or GPL is the way to go, not a self-created license. This is also for safety purposes, as the license has not been reviewed by legal experts. Additionally, it has typos.

**Comment by hello-smile6** [comment]
> Please use a normal license, or I will not give permission for my logo to be used with Catalyst. The MIT or GPL is the way to go, not a self-created license. This is also for safety purposes, as the license has not been reviewed by legal experts. Additionally, it has typos.

Lemme look at those typos, I'm in the market for some misspelled words!

**Comment by hello-smile6** [comment]
> Please use a normal license, or I will not give permission for my logo to be used with Catalyst. The MIT or GPL is the way to go, not a self-created license. This is also for safety purposes, as the license has not been reviewed by legal experts. Additionally, it has typos.

Could you point out other typos? I found the "warranty" one, but are there any others?

**Comment by hello-smile6** [comment]
> Please use a normal license, or I will not give permission for my logo to be used with Catalyst. The MIT or GPL is the way to go, not a self-created license. This is also for safety purposes, as the license has not been reviewed by legal experts. Additionally, it has typos.

Thanks for helping with https://github.com/SpellCheckJS/SpellCheckJS/commit/e265e5725211d706d60a99352f506b63ca2fc034

**Comment by icarusinthewings** [comment]
@JaydenDev

**Comment by imdevarsh** [comment]
Yeah, a normal license is more safe and definitely legal. MIT also has similar standards to this one, from what I can tell.

**Comment by icarusinthewings** [comment]
Resloved in b0a80bd.
